executor/executor.cc

```diff
@@ -601,12 +601,16 @@ void execute_one()
 			prog_extra_cover = true;
 			call_extra_cover = true;
 		}
-		if (strcmp(syscalls[call_num].name, "syz_usb_connect") == 0) {
+		if (strncmp(syscalls[call_num].name, "syz_usb_connect", 15) == 0) {
 			prog_extra_timeout = 2000;
 			// Must match timeout in pkg/csource/csource.go.
 			call_extra_timeout = 2000;
 		}
-		if (strcmp(syscalls[call_num].name, "syz_usb_disconnect") == 0) {
+		if (strncmp(syscalls[call_num].name, "syz_usb_control_io", 18) == 0) {
```

Comment:
s/18/strlen("syz_usb_control_io")/
I don't want to count chars in these strings to verify that they are 15 and 18 chars respectively each time I am reading this code.

Reply:
Done.